doc/nrf/releases_and_maturity/releases/release-notes-2.5.3.rst

```diff
@@ -0,0 +1,140 @@
+
+Supported modem firmware
+************************
+
+See `Modem firmware compatibility matrix`_ for an overview of which modem firmware versions have been tested with this version of the |NCS|.
+
+Use the latest version of the nRF Programmer app of `nRF Connect for Desktop`_ to update the modem firmware.
+See :ref:`nrf9160_gs_updating_fw_modem` for instructions.
+
+Modem-related libraries and versions
+====================================
+
+.. list-table:: Modem-related libraries and versions
+   :widths: 15 10
+   :header-rows: 1
+
+   * - Library name
+     - Version information
+   * - Modem library
+     - `Changelog <Modem library changelog for v2.5.3_>`_
+   * - LwM2M carrier library
+     - `Changelog <LwM2M carrier library changelog for v2.5.3_>`_
+
+Known issues
+************
+
+Known issues are only tracked for the latest official release.
+See `known issues for nRF Connect SDK v2.5.3`_ for the list of issues valid for the latest release.
+
+.. _ncs_release_notes_253_changelog:
+
+Changelog
+*********
+
+The following sections provide detailed lists of changes by component.
+
+Protocols
+=========
+
+This section provides detailed lists of changes by :ref:`protocol <protocols>`.
+
+Bluetooth
+---------
+
+Fixed an issue where the Bluetooth host running on the nRF5340 application core might deadlock on disconnection (DRGN-21390).
+
+Wi-Fi
+-----
+
+* Added:
+
+ * Build-time disconnection timeout configuration.
+ * The following fixes and improvements:
+
+   * Wi-Fi stuck in an awake state.
+   * Frequent Wi-Fi disconnections in a congested environment and low RSSI.
+   * https upload and download performance issues.
+   * MQTT disconnection in a congested environment and low RSSI.
+   * Selecting the best RSSI access point.
+   * Rate adaptation enhancements to improve range in 2.4 GHz band.
+
+ To accommodate all fixes and improvements, there is a 1 kB increase in the stack usage of WPA supplicant and, user must increase the stack sizes appropriately in their applications.
+
+* Updated WPA supplicant to reserve libc heap memory rather than using leftover RAM.
+  This does not affect the overall memory used, but now the RAM footprint as reported by the build will show higher usage.
+
+Matter
+------
+
+* Updated:
+
+  * The default Message Reliability Protocol (MRP) retry intervals for Thread devices to two seconds to reduce the number of spurious retransmissions in Thread networks.
+  * Increased the number of available packet buffers in the Matter stack to avoid packet allocation issues.
+
+Samples
+=======
+
+This section provides detailed lists of changes by :ref:`sample <samples>`.
+
+Matter samples
+--------------
+
+* :ref:`matter_lock_sample` sample:
+
+  * Added support for emulation of the nRF7001 Wi-Fi companion IC on the nRF7002 DK.
+
+
+Zephyr
+======
+
+The Zephyr fork in |NCS| (``sdk-zephyr``) contains all commits from the upstream Zephyr repository up to and including ``a768a05e6205e415564226543cee67559d15b736``.
```

Comment:
@cvinayak  to confirm the commit here.

Reply:
confirmed.

> git merge-base upstream/main HEAD
a768a05e6205e415564226543cee67559d15b736